edenai_apis/tests/features/image/test_amazon_image_embeddings.py

```
@@ -0,0 +1,325 @@
+        # Verify the bedrock client was called
+        amazon_image_api.clients["bedrock"].invoke_model.assert_called_once()
+
+    def test_embeddings_with_dimension_256(
+        self, amazon_image_api, sample_image_file, mock_bedrock_response
+    ):
+        """Test embedding generation with 256 dimensions."""
+        amazon_image_api.clients["bedrock"].invoke_model = MagicMock(
+            return_value=mock_bedrock_response
+        )
+
+        result = amazon_image_api.image__embeddings(
```

Ruff (0.14.3)

Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)